lib/fake_braintree/credit_card_serializer.rb

```diff
@@ -0,0 +1,20 @@
+module FakeBraintree
+  class CreditCardSerializer
+    def initialize(credit_card)
+      @credit_card = credit_card
+    end
+
+    def to_h
+      last_2 = @credit_card.last_4[-2..-1]
+      card_type = @credit_card.card_type
+      {
+        type: 'CreditCard',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.